### PR TITLE
NDRS-464--nctl-fold

### PR DESCRIPTION
**Pull request by ghost**
1. Created a new sub-directory named utils into which nctl library was placed.
2. Updated .gitignore so as to avoid committing nctl generated assets.
3. Renamed $NCTL/docs/installation.md -> $NCTL/docs/setup.md.

### CODE REVIEW

File: utils/nctl/docs/setup.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+make setup-rs
+```
+
+### Step 2 - set local setup file.
+
+```
+cat >> $HOME/.casper-nctl <<- EOM
+
+# Activate nctl shell.
+. $(pwd)/casper-node/utils/nctl/activate
```
Comment (Fraser999) [inline utils/nctl/docs/setup.md:25]: Will that be correct if the user has done `cd YOUR_WORKING_DIRECTORY/casper-node` in the step above?

File: utils/nctl/docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+```
+nctl-assets-teardown
+```
+
+Not only does this delete all previously created assets, it also stops any nodes still running in daemon mode.
+
+## Summary
+
+Using nctl one can spin up either a single or multiple test networks.  Each network is isolated in terms of its assets - this includes port numbers.  The nctl commands parrameter defaults are set for the general use case of testing a single local 5 node network.  You are encouraged to integrate nctl into your daily workflow so as to standardise the manner in which the network is tested in a localised setting.
```
Comment (Fraser999) [inline utils/nctl/docs/usage.md:131]:
```suggestion
Using nctl one can spin up either a single or multiple test networks.  Each network is isolated in terms of its assets - this includes port numbers.  The nctl commands parameter defaults are set for the general use case of testing a single local 5 node network.  You are encouraged to integrate nctl into your daily workflow so as to standardise the manner in which the network is tested in a localised setting.
```

**Review by Fraser999 — COMMENTED** [review]
Can we remove `run-dev.sh` and `stop-dev.sh` in this PR too?

Also, it might be worth changing [the client's default port](https://github.com/Fraser999/casper-node/blob/master/client/src/common.rs#L21) to `50101`?

File: utils/nctl/docs/usage.md (hunk 2 above)
Comment (momipsl) [reply utils/nctl/docs/usage.md:131]: My laptop keyboard has an issue whereby half the time I type the letter 'r' it types 'rr'.

File: utils/nctl/docs/setup.md (hunk 1 above)
Comment (momipsl) [reply utils/nctl/docs/setup.md:25]: 1.  I just corrected this potential path issue. 
2. I am inclined **not** to include deprecating `run-dev.sh` & `stop-dev.sh` are in this PR too.

**Review by Fraser999 — APPROVED** [review]

**Pull request closed without merging**